src/com/ecfeed/core/generators/algorithms/NwiseScoreEvaluator_Rep.java

```diff
@@ -0,0 +1,151 @@
+package com.ecfeed.core.generators.algorithms;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import com.ecfeed.core.generators.api.GeneratorException;
+import com.ecfeed.core.generators.api.IConstraintEvaluator;
+import com.ecfeed.core.utils.EvaluationResult;
+
+public class NwiseScoreEvaluator_Rep<E> implements IScoreEvaluator<E> {
+
+	private final Map<List<E>, Integer> fScores = new HashMap<>(); // store all the scores
+	private final Map<List<E>, Integer> fTupleOccurences = new HashMap<>(); // store all the constructed tables
+	private static final int NOT_INCLUDE = -1;
+	private int argN; // The number of dimensions to be covered
```

Comment:
make the names consistent internally, but also with the convention that is used in other files. 
fArgN
fDimensionCount
fConstraintEvaluator

Reply:
OK. Will make them consistent.